weave/ops_domain/artifact_version_ops.py

```diff
@@ -199,6 +199,22 @@ def metadata(
     wdt.ArtifactCollectionType,
 )
 
+gql_direct_edge_op(
```

Comment:
given the refactor of https://github.com/wandb/core/pull/16911, i think these need to be changed to `gql_connection_op(`